config.example.toml

```diff
@@ -1,2 +1,2 @@
-#gcc-path = "gcc-build/gcc"
+#gcc-path = "gcc-install/lib"
```

Comment:
Please revert. I never run `make install` and I'd like to keep it that way. You can mention the install folder too though.

Reply:
I believe `make install` can be necessary for e.g. using LTO.